core/src/test/scala/unit/kafka/server/SaslApiVersionsRequestTest.scala

```diff
@@ -51,7 +51,7 @@ object SaslApiVersionsRequestTest {
 
     // Configure control plane listener to make sure we have separate listeners for testing.
     val serverProperties =  new java.util.HashMap[String, String]()
-    serverProperties.put(SocketServerConfigs.CONTROL_PLANE_LISTENER_NAME_CONFIG, controlPlaneListenerName)
+//    serverProperties.put(SocketServerConfigs.CONTROL_PLANE_LISTENER_NAME_CONFIG, controlPlaneListenerName)
```

Comment:
I guess we can cleanup this class as well?

Reply:
Here is a PR for this goal.

Reply:
Can you remove the commented code?

Reply:
Remove.